src/main.py

```diff
@@ -1,11 +1,25 @@
-import os
+"""
+Script to determine leader and follower roles.
+
+Takes input from the camera to determine if the current vehicle
+should lead or follow based on whether or not there exists an ARTag
+in front of the vehicle.
+
+If an ARTag exists, the vehicle will become a follower vehicle.
+If an ARTag does not exist, the vehicle will become the leader vehicle.
+"""
 
 from follower import Follower
 from leader import Leader
 from tagrec import TagRecognition
 
+from bluetoothctl import *
```

Comment:
this would be `from bluetoothctl import Bluetoothctl`

but again, it's in my other branch right now.